linkerd/meshtls/rustls/src/creds/verify.rs

```diff
@@ -0,0 +1,97 @@
+use linkerd_identity as id;
+use std::convert::TryFrom;
+use std::time::SystemTime;
+use std::{io, sync::Arc};
+use tokio_rustls::rustls::{
+    self,
+    client::{self, ServerCertVerified, ServerCertVerifier},
+    server::ParsedCertificate,
+    Certificate, RootCertStore, ServerName,
+};
+use tracing::trace;
+
+pub(crate) struct AnySanVerifier(Arc<RootCertStore>);
+
+impl AnySanVerifier {
+    pub(crate) fn new(roots: impl Into<Arc<RootCertStore>>) -> Self {
+        Self(roots.into())
+    }
+}
+
+impl ServerCertVerifier for AnySanVerifier {
```

Comment:
Presumably this logic originated somewhere else? Can we reference that source concretely and specifically? Are there any copyright disclaimers we should make?

Reply:
Added a comment, not sure about disclaimers... This is licensed under Apache. Should we mention something?